### ISSUE TITLE
Bonjour issues masquerading as Homebridge Stability issues

### ISSUE THREAD

**Issue by JangoBritt** [nfarina/homebridge#862]
Hi all.

This isn't an issue as such, just something I've discovered, that I thought I would share.

I've been plagued with stability issues of my homebridge since day one. The issue was that even though everything looked fine on the Mac, very frequently, the homebridge devices would show up on the phone as Not Responding. I've done so many different things to debug this, that I'd really almost given up on using homebridge. 

Seeing everyone else's success however, spurred me into action. I did some more digging, and it appears that my ISP Router (BT HomeHub 5) has very wide-spread issues with Bonjour (mDNS). Recommendations were to switch/rename off the 2.4GHz network, and try to get all devices routing through the same 5GHz Radio. The problem with that, was that my Mac Mini was wired, so there was no way to tell Homebridge to only use the WiFi connection to serve the bonjour connection, and not the wired connection.

I finally identified this as a common issue, because I also installed an Airport Express in my home, which would lose discovery almost daily.

I took the nuclear option. I went out and bought a second hand Airport Extreme WiFi router from eBay. Wired everything up to go through it, and now not only has my airport express not lost connection, but my Homebridge hasn't once been unreachable.

in Summary, if you're having issues with homebridge devices showing up as "Not Responding," it may be an issue with your ISP's WiFi router.

**Comment by kwithus** [comment]
As I do have a "no response" issue in home in conjunction with Homebridge I would like to get some additional info.
When your devices showed as with no response could you actually access them or not? Mine show up as no response, however clicking on them does show the status and they do react on commands. When I leave home and open again, they show again as no response.

**Comment by Samfox2** [comment]
@kwithus thats exactly the same behaviour that my homebridge shows.

**Comment by kwithus** [comment]
Well, hopes you would give a different answer ;-)
Because I did change my setting to have Apple TV 4 connected to an Airport express which also now is the wifi Source for my house (together with an time capsule, which serves the 2 floor and have the raspberry Pi where FHEM and Homebridge sits on connected by LAN to the express directly. Asetzing which I think is similar to you solution.
But still the same strange behavior here
Any idea?
